exchanges/bybit/bybit_test.go

```diff
@@ -3537,3 +3539,49 @@ func TestUpdateOptionsTickerInformation(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestGetOpenInterest(t *testing.T) {
+	t.Parallel()
+	_, err := b.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  currency.ETH.Item,
+		Quote: currency.USDT.Item,
+		Asset: asset.Spot,
+	})
+	assert.ErrorIs(t, err, asset.ErrNotSupported)
+
+	resp, err := b.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  usdcMarginedTradablePair.Base.Item,
+		Quote: usdcMarginedTradablePair.Quote.Item,
+		Asset: asset.USDCMarginedFutures,
+	})
+	assert.NoError(t, err)
+	if assert.NotEmpty(t, resp) {
+		assert.NotZero(t, resp[0].OpenInterest)
+	}
+
+	resp, err = b.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  usdtMarginedTradablePair.Base.Item,
+		Quote: usdtMarginedTradablePair.Quote.Item,
+		Asset: asset.USDTMarginedFutures,
+	})
+	assert.NoError(t, err)
+	if assert.NotEmpty(t, resp) {
+		assert.NotZero(t, resp[0].OpenInterest)
+	}
+
+	resp, err = b.GetOpenInterest(context.Background(), key.PairAsset{
+		Base:  inverseTradablePair.Base.Item,
+		Quote: inverseTradablePair.Quote.Item,
+		Asset: asset.CoinMarginedFutures,
+	})
+	assert.NoError(t, err)
+	if assert.NotEmpty(t, resp) {
+		assert.NotZero(t, resp[0].OpenInterest)
+	}
+
+	resp, err = b.GetOpenInterest(context.Background())
```

Comment:
For some reason this is not being set correctly when getting all open interest:

<img width="527" alt="image" src="https://github.com/thrasher-corp/gocryptotrader/assets/9391715/eb71f7aa-9440-4038-9320-b671d3c1a514">

Reply:
The struct below was empty 😄